grouper/fe/templates/email/user_tokens_changed_html.tmpl

```diff
@@ -0,0 +1,14 @@
+{% extends "email/base_html.tmpl" %}
+
+{% block subject %}User tokens changed{% endblock %}
+
+{% block content %}
+
+<p>User token {{ action }} for <strong><a href="{{url}}/users/{{changed_user}}">{{ changed_user }}</a></strong>.</p>
+
+{% if changed_user != actioner %}
+    <p>This edit was made by
+    <a href="{{url}}/users/{{actioner}}">{{ actioner }}</a>.</p>
+{% endif %}
+
```

Comment:
this will never happen, right? are you planning for administrative access to change these on another user's behalf?